server/views/applications/pages/assess-placement-risks-and-needs/placement-considerations/risk-management-plan.njk

```diff
@@ -16,6 +16,8 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
+      <input type="hidden" name="version" value="{{ page.latestVersion }}"/>
```

Comment:
Very nice approach. We can layer this approach into each question if/when we need to.